.ko.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Chainguard, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+baseImageOverrides:
+  # TODO(mattmoor): Talk to kaniini about the best image to use here.
+  chainguard.dev/apko/cmd/apko: docker.io/alpine:latest
```

Comment:
@kaniini any thoughts on the best image to use here?

We should be able to set up some actions to start publishing our own `apko`-based image here and self-host off of that pretty quickly 😈

Reply:
docker.io/alpine:latest seems fine for now